TestAccBucket and TestReleaseVerification fail to compile examples: S3Client.send not on type

## Summary

Two integration tests are failing because their example programs no longer compile:

- `TestAccBucket` → `examples/bucket/index.ts:47`
- `TestReleaseVerification` → `examples/release-verification/index.ts:43`

Both fail with the same error:

\`\`\`
error TS2339: Property 'send' does not exist on type 'S3Client'.
\`\`\`

## Root cause

Both `examples/bucket/package.json` and `examples/release-verification/package.json` pin `@types/node: ^8.0.0`, which is far too old for the modern `@aws-sdk/client-s3` v3 type definitions that depend on newer Promise/AsyncIterable types from `@types/node`.

## Reproduction

Verified failing on master at sha `590ff281fc`:
- TestAccBucket: https://github.com/pulumi/pulumi-aws/actions/runs/24181743345/job/70592034644
- TestReleaseVerification: https://github.com/pulumi/pulumi-aws/actions/runs/24181743345/job/70592034629

Surfaced while reviewing #6266.

## Fix

Bump `@types/node` (and likely other ancient deps) in both example `package.json` files to a current version, then re-enable both tests (currently skipped via `t.Skip` referencing this issue).

## Related

- Skipped via #6266 to unblock that PR (the failures are unrelated to the r8 instance-type changes).

--- a/examples/examples_nodejs_test.go
+++ b/examples/examples_nodejs_test.go
@@ -1438,7 +1438,6 @@ func TestAccConfigureClusterAddons(t *testing.T) {
 
 // also test the release verification test so it will also fail in CI
 func TestReleaseVerification(t *testing.T) {
-	t.Skip("temporarily disabled, see https://github.com/pulumi/pulumi-aws/issues/6272")
 	region := getEnvRegion(t)
 	test := integration.ProgramTestOptions{
 		Dir: filepath.Join(getCwd(t), "release-verification"),

--- a/examples/release-verification/package.json
+++ b/examples/release-verification/package.json
@@ -11,6 +11,7 @@
         "@pulumi/pulumi": "^3.0.0"
     },
     "devDependencies": {
-        "@types/node": "^8.0.0"
+        "@types/node": "^18.0.0",
+        "typescript": "^5.0.0"
     }
 }

